packages/apps/crm/src/components/organisms/Prospect/ProspectDropdownCards/ProspectDropdownCards.js

```diff
@@ -86,6 +95,16 @@ const ProspectDropdownCards = ({}) => {
             key: 4,
             childrenComp: <DropdownEventView eventList={listEventPartner} />,
           },
+          {
+            title: I18n.t('Crm_Opportunity'),
+            key: 5,
+            childrenComp: (
+              <DropdownOpportunityView
+                opportunityList={prospectOpportunityList}
+                opportunityStatusList={opportunityStatusList}
```

Comment:
opportunityStatusList can be added directly in DropdownOpportunityView using the selector.